Simulations/python/runRecipes.py

```diff
@@ -109,6 +114,11 @@ def parseCommandLine(self,args):
         params['testRun'] = args.testRun
 
         params['calibFile'] = args.calibFile
+
+        if(args.nCores):
```

Comment:
You don't need parenthesis here:
```suggestion
        if args.nCores:
```
(note that you can just accept suggestions like this directly from the github page)